### ISSUE TITLE
Add this telnet server example I wrote to the Conch examples

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3940]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3940|
|Type|enhancement|
|Created|2009-07-29 15:58:37Z|
|Branch|https://github.com/twisted/twisted/tree/telnet-server-example-3940|

People often have trouble figuring out how the telnet server code works.  It's a bit different from a lot of the other things in Twisted, in that it involves explicitly application-level containment, rather than inheritance, to provide the higher-level interface.

When the most recent person to ask for help with this asked for help with this, I wrote the attached to help him get going in the right direction.  It would probably help other people, too, if they could see it.

Attachments:

* [telnet_echo.tac](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6f3/6f3a4ae830d368f3ab6bda2bc6a51673c43c8452/e1b08b874bdc3395169066c0e1f1c0448e268c20.tac) (1079 bytes) - added by exarkun on 2009-07-29 15:59:00Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3940 3940
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_telnet_server_example_3940 branches/telnet-server-example-3940
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__documentation documentation
time__1248883117000000 1248883117000000
changetime__1257692570000000 1257692570000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

I propose for this file to be added to `doc/conch/examples/` and linked from the `index.xhtml` in that directory.  Huh.  I guess I should just make a branch.  Sheesh.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [27205]) Branching to 'telnet-server-example-3940'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

Alright, I did.  Ignore the attached file, just look at the branch.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh set owner to @glyph|
|-|-|

Looks fine to me :-)

Checked for typos, didn't find any. Since this is pure documentation, shouldn't change anything in terms of tests. Tried `twistd -ny telnet_echo.tac` and worked nicely :-) 

I think this is an obvious candidate for inclusion in trunk. :-)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [27400]) Merge telnet-server-example-3940

Author: exarkun
Reviewer: lvh
Fixes: [#3940](https://github.com/twisted/twisted/issues/3940)

Add a simple example echo-telnet server to Conch.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
